docs/server/features/archiving_blog_post.md

```diff
@@ -0,0 +1,248 @@
+
+This mechanism is in place to allow reading data from the archive quickly: Having a constant suffix (`.000001`) and one chunk file per logical chunk allows any node to quickly find the chunk file containing a certain _log position_ in the archive.
+This has some (minor) implications: When you use _Archiving_, you cannot easily tell from a directory listing of the archive:
+
+- whether a chunk was scavenged or not
+- the number of times a chunk was scavenged (this information is local to a node anyway)
+
+After some time, all the chunks will eventually be uploaded to the archive.
+Great, now let's jump to the other nodes to see what's going on there!
+
+## Deleting archived data locally
+
+Doing a directory listing of one of the follower or leader nodes shows nothing special - all of the chunks are still here (there are only 6 chunks in total for this demo):
+```
+$ ls /var/lib/kurrentdb/
+chaser.chk           chunk-000002.000000  chunk-000005.000000  proposal.chk
+chunk-000000.000000  chunk-000003.000002  epoch.chk            truncate.chk
+chunk-000001.000000  chunk-000004.000000  index                writer.chk
+```
+
+Hmmm, so how do we get rid of the local chunks that are already in the archive? The answer is: _run a scavenge_!
+
+Let's try it out: we click on the _scavenge_ button on the UI, but...nothing seems to happen - all the chunks are still there. Hmmm, what's happening?
+Remember, we had a retention policy to keep chunks having data less than 30 days old!
+
+A quick peek at the logs shows that this was in fact the reason the chunks were kept:
+```
+[DBG] SCAVENGING: ChunkRemover is keeping chunk "#0-0 (chunk-000000.000000)"
+because of the retention policy for days
+[DBG] SCAVENGING: ChunkRemover is keeping chunk "#1-1 (chunk-000001.000000)"
+because of the retention policy for days
+```
+
+#### 30 days later...
+
+We run a scavenge again 30 days later and sure enough, some of the chunks (0, 1, 2 & 3) are deleted from the node!
+```
+$ ls /var/lib/kurrentdb/
+chaser.chk           chunk-000006.000001    index         writer.chk
+chunk-000004.000000  chunk-000007.000000    proposal.chk
+chunk-000005.000001  epoch.chk              truncate.chk
+```
+
+Notice that we had configured the archiving settings to keep only 500MB of data which should be roughly equivalent to 2 chunks but there are actually 4 chunks left.
+That's because running a scavenge (two times) had closed the chunks 5 & 6 with a _scavenge point_ and they were thus not full of data.
+
+You can now try to read some streams from a client or from the web UI, reads that go to the archive are handled transparently!
+
+## Conclusion
+
+This was a small introduction to _Archiving_.
+Many questions may have popped up in your head, for example:
+
+- How do I determine proper values for the retention policy?
+- How do I scavenge the archive?
+- How do backups work?
+
```

Comment:
Should we have a part 2 blog post that goes into these details?  I understand that they are in the documentation, but this may be useful content to also surface in another blog post as well.